platforms/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+app_name = 'platforms'
+urlpatterns = [
```

Comment:
na developie jest pokazane jak robić "ładne" url (sites) na bazie podejścia które wykorzystuje samo django wewnętrznie - pozwala to na używanie namespace'ów w nazwach url